saliency_metrics/metrics/insertion_deletion/insertion_deletion_result.py

```diff
@@ -0,0 +1,40 @@
+import os.path as osp
+from typing import Dict, List
+
+import mmcv
+import numpy as np
+
+from saliency_metrics.metrics.serializable_result import SerializableResult
+
+
+class InsertionDeletionResult(SerializableResult):
+    def __init__(
+        self,
+        summarized: bool = False,
+    ) -> None:
+
+        self.summarized = summarized
+        self.results: List[Dict[str, float]] = []
+
+    def add_single_result(self, single_result: Dict) -> None:
+        self.results.append(single_result)
+
+    def dump(self, file_path: str) -> None:
+        if self.summarized:
+            del_auc_scores_list: List[float] = []
+            ins_auc_scores_list: List[float] = []
+            for result in self.results:
+                del_auc_scores_list.append(result["del_auc"])
+                ins_auc_scores_list.append(result["ins_auc"])
+                summarized_result = {
```

Comment:
Why is a new `summarized_result` dict created inside the for-loop? and why are the mean and std computed insided the for-loop? The `summarized_result` should be moved to the outside, right?